libs/wlgen/wlgen/rta.py

```diff
@@ -144,7 +144,7 @@ def __postrun(self, params):
             return
         self._log.debug('Pulling logfiles to [%s]...', destdir)
         for task in self.tasks.keys():
-            logfile = "'{0:s}/*{1:s}*.log'"\
+            logfile = "{0:s}/*{1:s}*.log"\
```

Comment:
Good point. Actually, I'll use `self.target.path` for the sake of theoretical consistency.